### ISSUE TITLE
[Suggestion] Half Corner shape in block scale

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Skywanderers#1520]
The shape that was the result of issue #1135 and #1082 has proven to be extremely useful, this is just a request to bring the same shape to block scale. I recognise that it poses some problems with the current mirror tech but this would help to reduce the disparity in shape freedom between bricks and blocks

**Comment by tsunamayo** [comment]
Sorry to say this, but this shape is more likely to be removed as it is also causing collisions issues... Kinda suck I know :( . I am too trigger happy to implement things only to discover issue later on... Thanks

**Comment by ProPeach** [comment]
How certain is this? It would be a terrible shame to lose this shape, it's become almost essential to creating sloped edges without using hinges. It's often used on wings, spoilers, shells and other shapes that have previously needed a hinge to get a slope on

**Comment by ExodistSKY1** [comment]
I agree with Peach, I would like to see the shape kept as this is such a utilitarian brick that I have used on a lot of things to make my ships flow a lot smoother aesthetically.

**Comment by Garrett-C** [comment]
I also think it would be ideal if you could find a way to keep it. It is really important for making 2 slopes that smoothly connect without child entities or clipping. 

For example if you want to do a star wars style docking port that connects into a sloped surface like on the falcon or in this case the ghost.
![image](https://user-images.githubusercontent.com/43644633/69564677-5817cb00-0fab-11ea-9f49-1edfaffc127e.png)
If I wanted to achieve that smooth join without the half corner shape I would need to use child entities. So for this example, the block removes the need for an entire child entity.

There are also some smaller uses like here on my obsidian class where I used it to smooth out the area around where the controls will be going.
![image](https://user-images.githubusercontent.com/43644633/69564956-e12f0200-0fab-11ea-84a4-2c2b015a2dfb.png)

All in all this is one of the most useful bricks in the game both for making the game run better and adding those smoother features. I really do hope you can find a way to resolve the issues it has without removing it.

**Comment by Aiwii** [comment]
![pleasekeephalfcorner](https://user-images.githubusercontent.com/53021785/69566614-b7a6b400-0f74-11ea-943c-d0c638250364.gif)

Not really sure how to fill this gap _without a half corner_... It would be really nice if it could stay. Now that I know how to properly implement it it's **REALLY** useful when building. Particularly when adding extensions to sloped surfaces as Grarett-C stated. Please consider keeping it.

**Comment by PopkaAkula** [comment]
Though I personally have never used this brick nor found a way to use it, it will be a great detriment if it was removed from the game. Its just one of those shapes that can't be replicated in any other way and it wouldn't make sense to get rid of it. I hope there is a way for you to fix the collision issues rather than getting rid of the brick.

**Comment by SageThe13th** [comment]
Removal of the shape in question would be huge lose for the game. It's a shape I've found myself needing in other games frequently and now in Skywanders we actually have it. The only way to bypass using that shape would be child entities. A solution which presents it's own difficulties. Where as this shape if it can be saved is nice simple solution to a much encountered building problem that often requires drastic measures to address.

**Comment by ProPeach** [comment]
Here are some examples of where the slope is used. It's commonly found where two sloped section meet to form a compound edge.
![image](https://user-images.githubusercontent.com/27857711/69638623-f4e17380-1052-11ea-8e9d-7597373b9996.png)
![image](https://user-images.githubusercontent.com/27857711/69638659-062a8000-1053-11ea-82c1-dcd914bcf90e.png)
This example is what prompted me to ask for the block version of the shape, but of course its uses stretch far beyond this ship (you may recognise it from the end of Rogue One). Here it's being used to create a transition between a curved surface and a straight cut. Some builds like this require some careful planning, and shapes like these are incredibly rewarding to work with and can be used to create satisfying, unique shapes unlike any other game.
![image](https://user-images.githubusercontent.com/27857711/69638720-23f7e500-1053-11ea-9f27-126ae1056a1b.png)
